ocs_ci/helpers/dr_helpers_ui.py

```
@@ -915,13 +959,149 @@ def check_or_assign_drpolicy_for_discovered_vms_via_ui(
             acm_obj.take_screenshot()
-            expected_conf_msg = conf_msg.split("\n", 1)[1].strip()
-            assert expected_conf_msg == "New policy assigned to application"
+            assert (
+                "New policy assigned to application" in conf_msg
+            ), f"Unexpected confirmation message: '{conf_msg}'"
             log.info("Close page")
             acm_obj.do_click(acm_loc["close-page"], enable_screenshot=True)
     return True
 
 
+def remove_drprotection_for_discovered_vm_via_ui(
```

Ruff (0.15.20)

[warning] Too many statements (51 > 50)

(PLR0915)